customer-commons/src/main/java/no/unit/nva/customer/model/CustomerDao.java

```diff
@@ -66,6 +67,7 @@ public class CustomerDao implements Typed {
     private Sector sector;
     @JsonAlias("rightRetentionStrategy")
     private RightsRetentionStrategyDao rightsRetentionStrategy;
+    private List<PublicationTypes> publicationTypes;
```

Comment:
I think the naming is a bit off here. It becomes unclear what it is. `allowFileUploadForPublicationInstances`, or simply `allowFileUploadFor` would probably be better.